File: NetFramework/TouchToolkit/CS/TouchToolkitDemo/GcZoomPages/Help UserControl/PanWindowControlBar.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace ClickOnceDemo.C1ZoomPages.Help_Forms
{
    public partial class ControlBarUserControl01 : UserControl
    {
        public ControlBarUserControl01()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            this.C1ZoomCommandProvider1.OwnerForm.Close();
        }
    }
}

File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/Help UserControl/PanWindowControlBar.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace C1ZoomPages.Help_Forms
	Public Partial Class ControlBarUserControl01
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(sender As Object, e As EventArgs)
			Me.gcZoomCommandProvider1.OwnerForm.Close()
		End Sub
	End Class
End Namespace
